### PR TITLE
Styled home page list of list names

### PR DESCRIPTION
**Pull request by sillytsundere**
## Description

Styled `Home` page lists. Each list is now a link which when clicked takes the user to the `List` page for that list.

## Type of Changes

styling

### Before

Screenshot 2024-04-01 at 6 18 20 PM

### After

Screenshot 2024-04-01 at 6 17 10 PM
Screenshot 2024-04-01 at 6 17 01 PM

## Testing Steps / QA Criteria

- `git pull`
- 'git checkout pc-style-home-list
- `npm ci`
- `npm run start`
- navigate to the `Home` page and view the list of List names
- click a list name to be navigated to the list page for that list

### CODE REVIEW

**Review by rachelspencer — APPROVED** [review]
Looks really nice @sillytsundere! I tested out the list links in both mobile and desktop and all looks/works well!

I just finished styling the `List` page and I altered the text decoration to none for the `.list-title` css rule. I thought it looked more modern. Which do you prefer?

**Comment by sillytsundere** [comment]
@rachelspencer I agree I think it looks better without the undeeline, I can edit that in this branch for the home page!

**Review by trushmi — APPROVED** [review]
Nicely done @sillytsundere !

**Review by jaguilar89 — APPROVED** [review]
Good job @sillytsundere!

File: src/components/SingleList.css [hunk 1]
```diff
@@ -28,11 +28,10 @@
...
-.SingleList > Link {
-	margin-left: 0.2em;
-	width: 100%;
-	display: flex;
-	justify-content: space-between;
+.list-link {
+	margin-left: 1em;
+	color: var(--color-text);
+	text-decoration: none;
 }
```
Comment (polly89) [inline src/components/SingleList.css:35]: This works great for light mode, but it isn't visible in dark mode. Are you all planning on handling dark mode in a separate issue? I don't want to ask you to change the text color, if the background color is going to change. Currently looks like:
Screenshot 2024-04-03 at 2 12 40 PM

**Review by polly89 — CHANGES_REQUESTED** [review]
The list-links look and function beautifully in light mode. I left a comment regarding dark mode. Let me know how the team plans to approach it.

File: src/components/SingleList.css (hunk 1 above)
Comment (sillytsundere) [reply src/components/SingleList.css:35]: @polly89 The background color is going to change for dark mode. Right now that issue is related to what Jose has been working with in the dark-light toggle PR and we are planning to assess it once all our current PR's have been merged so we can assess what still needs to be done.

Comment (polly89) [reply src/components/SingleList.css:35]: Sounds good. I'll approve this PR.

**Review by polly89 — APPROVED** [review]

**Pull request merged**